horovod/runner/mpi_run.py

```diff
@@ -153,7 +153,7 @@ def mpi_run(settings, nics, env, command, stdout=None, stderr=None):
     if mpi_impl_flags is None:
         raise Exception(_MPI_NOT_FOUND_ERROR_MSG)
 
-    impi = _IMPI_IMPL == mpi
+    impi = (_IMPI_IMPL == mpi or _MPICH_IMPL == mpi)
```

Comment:
Can this logic be rephrased to this, which I think is more readable?
```suggestion
    impi = mpi in (_IMPI_IMPL, _MPICH_IMPL)
```

Reply:
I think, `impi` should be renamed to `impi_or_mpich` as this is what it now encodes.

Reply:
Fixed.